runway/cfngin/lookups/handlers/kms.py

```diff
@@ -73,4 +74,7 @@ def handle(cls, value, context=None, provider=None, **kwargs):
         decoded = codecs.decode(value, "base64")
 
         # decrypt and return the plain text raw value.
-        return kms.decrypt(CiphertextBlob=decoded)["Plaintext"]
+        decrypted = kms.decrypt(CiphertextBlob=decoded)["Plaintext"]
+        if sys.version_info[0] > 2:  # TODO remove after droping python 2
+            return decrypted.decode("UTF-8")
+        return decrypted
```

Comment:
could do something like this to remove the version check and instead rely on the data's type since it should always return a string.

```suggestion
        return decrypted.decode() if isinstance(decrypted, bytes) else decrypted
```

`UTF-8` can also be removed as its the default.